internal/core/metadata/application/notify.go

```diff
@@ -84,6 +92,8 @@ func deleteDeviceCallback(ctx context.Context, dic *di.Container, device models.
 	if response.StatusCode != http.StatusOK {
 		lc.Errorf("fail to invoke device service callback for deleting device %s, err: %s", device.Name, response.Message)
 	}
+
+	sendNotification(ctx, dic, device.Name, http.MethodDelete)
```

Comment:
Use `Device removal` instead of `http.MethodDelete`

Reply:
Modified.